Commenter.Extension/Properties/AssemblyInfo.cs

```diff
@@ -43,3 +43,11 @@
     AssemblyName = "Tvl.VisualStudio.Text.Commenter.Interfaces",
     Version = "1.0.0.0",
     CodeBase = "$PackageFolder$\\Tvl.VisualStudio.Text.Commenter.Interfaces.dll")]
+[assembly: ProvideCodeBase(
```

Comment:
:memo: This fixes a bug caused by an oversight in f46b6da371fc67bcec83e67a51228486de784a50.